lib/models/patch/builder.js

```diff
@@ -5,27 +5,73 @@ import File, {nullFile} from './file';
 import Patch from './patch';
 import {Unchanged, Addition, Deletion, NoNewline} from './region';
 import FilePatch from './file-patch';
+import MultiFilePatch from './multi-file-patch';
 
-export default function buildFilePatch(diffs) {
+export function buildFilePatch(diffs) {
```

Comment:
Do we still actually need `buildFilePatch()`? I believe `buildMultiFilePatch()` supersedes everything it can do.

I suppose the only benefit to keeping both is that `buildFilePatch()` enforces that we're expecting to receive exactly zero, one, or two diffs?

Reply:
Maybe? I'm fine just using `buildMultiFilePatch()`. From what I can tell we just need to handle the `emptyDiffFilePatch` case in it. And we'd need to consolidate the tests and see about which `buildFilePatch` tests need to be converted to `buildMultiFilePatch` tests. Makes the code cleaner and potentially reduces friction to change in the future. I'm 👍 if you wanna do this. I'm also fine leaving it for now and opening up a code cleanup issue for it